### ISSUE TITLE
support for React 18.2

### ISSUE THREAD

**Issue by g5-freemen** [DanteCoder/react-resizableBox#4]
Maybe it possible to fix this error:

npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^17.0.2" from @dantecoder/react-resizablebox@1.2.0
npm WARN node_modules/@dantecoder/react-resizablebox
npm WARN   @dantecoder/react-resizablebox@"^1.2.0" from the root project

**Comment by g5-freemen** [comment]
If you wish i can try to fix it by myself

**Comment by DanteCoder** [comment]
> @DanteCoder 
> If you wish i can try to fix it by myself

Yeah, please try updating the peer dependency of React. And thank you.

**Comment by g5-freemen** [comment]
Plz give me access to push updated package.json & package-lock.json or give me your email (to send this files to you)

**Comment by DanteCoder** [comment]
@g5-freemen I think you can fork this repository and then make a pull request to this repo.
https://github.com/gabrieldemarmiesse/getting_started_open_source

**Comment by g5-freemen** [comment]
@DanteCoder made PR
